test/models/project_test.rb

```diff
@@ -238,7 +238,27 @@
       output.string.include?('using the error callback').must_equal(true)
       output.string.include?("Can't get here").must_equal(false)
     end
-
   end
 
+  describe 'deploy group releases' do
```

Comment:
using `describe "#last_release_by_deploy_group"` makes it easier to see what method this is for